### DISCUSSION TITLE
Permissions required for access token to integrate JFrog in a template

### DISCUSSION THREAD

**Discussion by Conor-Behard333** [coder/coder#10213] [General] [answered]
Hey, I'm wanting to integrate an on prem instance of jfrog into a template and following these docs (https://coder.com/docs/v2/latest/platforms/jfrog) it states that I need an "admin-level access token for Artifactory" what are the required permissions needed for the access token in order for the integration to work (i.e. I don't want to give it "god mode" if its not required)

**Comment by matifali** [comment, marked as answer]
I am not an expert for artifactory but. It should be able to generate tokens for other users.
